lib/winpki/ldap.go

```diff
@@ -100,34 +139,14 @@ const searchPageSize = 1000
 // is closed. Callers should check for trace.ConnectionProblem errors
 // and provide a new client with [SetClient].
 type LDAPClient struct {
```

Comment:
Is this struct even necessary anymore? It's not really a client at all - it only holds configuration data.

I also think the godoc needs to be updated. The whole idea of having to call `SetClient` to refresh it goes away, right?

Reply:
It isn't needed, I've removed that struct and have moved its functions to the LDAPConfig struct